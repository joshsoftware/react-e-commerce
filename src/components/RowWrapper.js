import React from 'react';
import { Row } from 'reactstrap';
import PropTypes from 'prop-types';

const RowWrapper = ({ data, noGutters, form, xs, sm, md, lg, xl }) => {
  return (
    <Row noGutters={noGutters} form={form} xs={xs} sm={sm} md={md} lg={lg} xl={xl}>
      {data}
    </Row>
  );
};

export default RowWrapper;

RowWrapper.propTypes = {
  data: PropTypes.oneOfType([PropTypes.element, PropTypes.array]),
  noGutters: PropTypes.bool,
  form: PropTypes.bool,
  xs: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  sm: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  md: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  lg: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  xl: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
};
