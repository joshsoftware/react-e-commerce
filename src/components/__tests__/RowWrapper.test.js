import React from 'react';
import { render } from '@testing-library/react';
import RowWrapper from '../RowWrapper';

describe('RowWrapper Component', () => {
  it('Must return row', () => {
    const { asFragment } = render(<RowWrapper />);

    expect(asFragment()).toMatchSnapshot();
  });
});
