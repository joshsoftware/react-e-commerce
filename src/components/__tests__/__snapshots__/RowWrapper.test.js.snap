// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowWrapper Component Must return row 1`] = `
<DocumentFragment>
  <div
    class="row"
  />
</DocumentFragment>
`;
